scripts/.env

```diff
@@ -0,0 +1,3 @@
+# Replace sample private key for an active validator address
+# For local testing, use accounts[1] private key from truffle
+ETHEREUM_PRIVATE_KEY="ae6ae8e5ccbfb04590405997ee2d52d2b330726137b875053c36d94e974d162f"
```

Comment:
@juniuszhou Do we need this ?

Reply:
This is for the smart contract testing as I understand it. However it this also under `./testnet-contracts` so I am reviewing this one.